src/main/java/dan200/computercraft/shared/turtle/core/TurtleMoveCommand.java

```diff
@@ -14,9 +14,9 @@
 import dan200.computercraft.shared.util.WorldUtil;
 import net.minecraft.block.Block;
 import net.minecraft.entity.Entity;
+import net.minecraft.util.EnumFacing;
```

Comment:
Why was this line moved? I assume the move has no effect and just makes the commit a bit messy.

Reply:
Because my IDE likes to put things in alphabetical order. I'm going to be honest, I can't really muster the energy to fix it.